### ISSUE TITLE
[wallet] fixes

### ISSUE THREAD

**Issue by florianlenz** [Bit-Nation/BITNATION-Pangea-mobile#402]
## Feature / Issue
I would like to request a few wallet changes. 

__Wallet:__ 
1. The address is hidden on a small screen when I try to enter it (iphone SE)
2. There should be no auto complete for addresses. 

__confirm screen__
1. There should be an indication on how fast / slow a transaction will be when moving the slider.
2. The amount we confirm should be displayed
3. The from and to address should be displayed.
4. The gas limit should be displayed AND should be changeable.
5. The transaction fee should be displayed (change when moving the slider)
6. The confirm screen has to accept a parameter called "purpose" and an "App" parameter. We need that one for e.g. when we try to sign a smart contract. If I e.g. use the send DApp the app name would be "Send DApp" and the purpose would be "send money to your friend" or what ever.

__in general__
- The wallet state should be persisted so that the wallet display your amount offline. We are supposed to be offline first. 
- when sending money the amount should be deducted from the amount you have in your wallet imminently. 
- It would be nice if we could use [pull down](https://facebook.github.io/react-native/docs/refreshcontrol.html) to refresh the wallet manually. 

## Acceptance criteria
- All listed points above need to be addressed.

**Comment by florianlenz** [comment]
For persisting the wallet I would a data structure that might look like this: 
```js
{
   symbol: "ETH"
   name: "Ethereum",
   ethereumBased: true,
   decimals: 18,
   balance: "00010000000000" 
}
```

**Comment by florianlenz** [comment]
And the amount should be persited with the lowes decimal

**Comment by albertoestarrona** [comment]
1 day. Friday

**Comment by florianlenz** [comment]
done. Closed in PR #420
